README.org

```diff
@@ -84,7 +84,7 @@ The =string-inflection-all-cycle=, conversion often. However,
 conversion occurs even when there is no need to lower-camelcase.
 Therefore so difficult to use, is not recommended
```

Comment:
Is this comment still accurate after https://github.com/akicho8/string-inflection/commit/4df4b62666991d6f57fe8968b20a8f938bade888?

Reply:
Accurate.
I still do not use `string-infection-all-cycle`.
It is better to use the appropriate function to match the emacs major-mode.

Reply:
>  It is better to use the appropriate function to match the emacs major-mode.